src/api/event/controllers/event.js

```diff
@@ -1,9 +1,38 @@
-'use strict';
+const { createCoreController } = require('@strapi/strapi').factories;
 
-/**
- * event controller
- */
+module.exports = createCoreController('api::event.event', ({ strapi }) => ({
+  async find(ctx) {
```

Comment:
Does this mean we only overwrite the find method? Other methods like update are still using strapi's default implementation?